proteomic_aggregation_and_visualization/python/proteomic_aggregation_and_visualization.ipynb

```diff
@@ -0,0 +1,4004 @@
+{
```

Comment:
<p>I think before you get into the "steps" it would help immensely to have some narrative here like "A challenge with proteomics data is that comparing between samples requires careful steps because of blah blah blah." </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microbiomedata/nmdc_notebooks/pull/120/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>